### PR TITLE
Bug 1280613 - Async append/remove for SourceBuffer

### PR DESCRIPTION
**Pull request by a2sheppy**
Add BCD info for SourceBuffer methods appendBufferAsync()
and removeAsync().

### CODE REVIEW

**Comment by a2sheppy** [comment]
Whichever API I copied my JSON from (I don't remember which) needs updating too then. My ordering and the lack of the value_to_set property carried over from that. Working on fixing now.

File: api/SourceBuffer.json [hunk 1]
```diff
@@ -1105,6 +1105,63 @@
...
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "62",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "media.mediasource.experimental.enabled"
```
[the remark this answers is not in the document]
Comment (ddbeck) [reply api/SourceBuffer.json]: Still missing the "value_to_set" here. Thanks for picking up everywhere else though!

File: api/SourceBuffer.json [hunk 2]
```diff
@@ -1274,6 +1353,68 @@
...
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "safari": {
```
Comment (ddbeck) [inline api/SourceBuffer.json]: Safari ought to come before Safari for iOS

Comment (a2sheppy) [reply api/SourceBuffer.json]: Thanks, good catches!

**Comment by ddbeck** [comment]
Excellent, thank you again @a2sheppy! 🎉

**Pull request merged**